### DISCUSSION TITLE
Stun Combats Funeral

### DISCUSSION THREAD

**Discussion by hal9000PR** [ParadiseSS13/Paradise#18179] [Projects]
## Here is a tracker for any PR that reworks a source of hard stuns.

if you wish to work on any of these, please DM me on discord, `hal 9000#2640`
when a PR is made for it I will replace the comment at the side with the PR number
you can code a PR for either of the categories 

## Current priorities, the most common stuns

- [x] Batons - #18127 
- [x] Antistun chems - #18084 
- [x] Antistun implants/organs - #18202
- [x] Flashbangs - #18189
- [x] Disarms - #18186
- [ ] Grabs - someone please refactor these for the love of god
- [x] Punch stun  - #18191
- [x] RNG brute weapon stun - #18212
- [x] Explosions - should cause knockdown + short stun, this is mostly because of guns that fire explosive rounds such as FRAG-12
- [x] Beepski + ED209 melee - #18232
- [x] Shocks - #18389
- [x] Help intent - #18360
- [x] Shotgun shells - #18287

## Lesser priorities, antag specific stuns

- [x] Cult stun hand - #18247 take one
- [ ] Thrown cult spear - The idea to make it embed and then the recall pull the person with the spear has been suggested before and sounds really cool
- [x] Magic missile - #18409
- [x] Vampire glare - #18524
- [x] Weighted bolas - #18207
- [x] Mech punches - #18405
- [x] Ebow - #18388
- [x] Chainsaw - #18252
- [x] Stun slugs for the shotgun - #18185
- [x] Krav leg sweep - #18404
- [ ] CQC - god help whoever tackles this one
- [x] sleeping carp - #18219
- [x] throwing people into stuff - #18363
- [x] smashing bottles over peoples heads - #18196
- [ ] C20-R - #18195, still needs a change
- [x] Meathook - #18255
- [x] compact sniper - #18357
- [x] flashes - #18275
- [x] defibs - #18456

**Comment by Inadvizable** [comment]
Door shocks? Those are pretty nasty and AI/borgs use them in combat.

**Comment by Bm0n** [comment]
Are you still going to make the stunprod bulky? i swear i remember talking to you about it

**Reply by hal9000PR** [reply]
I am up for the stunprod becoming bulkey and bringing its power inline with the stunbaton.

**Comment by GDNgit** [comment]
Cult spear was delt with in #19343 
CQC was (mostly) delt with in #19310
